docs/data/charts/bars/HorizontalBars.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { axisClasses } from '@mui/x-charts';
+
+const chartSetting = {
+  xAxis: [
+    {
+      label: 'rainfall (mm)',
+    },
+  ],
+  width: 500,
+  height: 300,
```

Comment:
What do you think about increasing the height of this demo?
IMHO, it looks very squished with the given dimensions... After all, the axes have been swapped. 🤔 
```suggestion
  height: 400,
```

Reply:
Yes, I always put 300 because otherwise it start taking too much space when scrolling, but this chart could get an exception :)